### DISCUSSION TITLE
Regarding the usage of my Github developer account for the purpose of uploading the technical projects that I have worked on as a pre requisite of getting offered a job in IT or software development in companies having projects on software development

### DISCUSSION THREAD

**Discussion by viksof** [community/community#59114] [Enterprise]
I am preparing for software development job interviews and I have to put some project details on which I have worked in my resume or Curriculum Vitae (CV) when applying to software development jobs in companies. 

I am seeing two options in the below mentioned website link for selecting for uploading the project files that I have developed on my Github account.

https://github.com/organizations/enterprise_plan

May I please know from among these, which plan should I choose for uploading the project details that I have developed to include it in my resume or Curriculum Vitae (CV) when applying to jobs in tech companies in the field of software development and how can I know more about it?

When I clicked on the 'Enterprise server' plan, it is leading me to a website whose link I am indicating below.

https://enterprise.github.com/trial?source=pricing-card-enterprise

It is indicating that it is only for a trial period of 45 days and I may have to pay money in order to continue using this account after the trial period.

However, I do not have any money with me to pay for the same.

May I please know how else can I get access to the resources needed to showcase my project details on which I have worked in order to put them in my resume or Curriculum Vitae (CV) while applying to jobs in companies?

Can you pls let me know about this?

**Comment by airtower-luna** [comment]
You don't need any kind of paid plan if you just want to have some public repositories. Just [create as many repositories as you need](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository) and push/upload your files. :slightly_smiling_face: 

If you're not familiar with Git yet, I strongly recommend looking at the Git Book: https://git-scm.com/book/en/v2

**Comment by viksof** [comment]
Hi,

Thank you for your email.

Thanks for the information.

May i please know how can I push/upload files on my Github repositories?

Can you pls let me know about this?

Thanks and best regards,

Vikas

**Comment by airtower-luna** [comment]
Please check the link to the Git Book I posted above, the first 3 chapters will teach you all the basics for working with Git repositories, including how to push to a remote repository (like on GitHub).

Alternatively you can also upload files over the web UI, but that's much more limited than using Git. And if you want to work in software development being able to work with Git (instead of just a web UI) is an important skill anyway. :wink:

**Comment by viksof** [comment]
Hi,

Thank you for your email.

Thanks for the information.

Isn't there any other contact email address of Github where I can check
about the usage of Github platform for uploading project files that I have
worked on in my Github account  in order to put them in my resume while
applying to jobs in software development?

Can you pls let me know about this?

Thanks and best regards,

Vikas

**Comment by airtower-luna** [comment]
Please mind that this is a public forum, not a private email exchange (even though you can reply by email), and I'm not a support employee, just another user trying to help. :slightly_smiling_face: 

A repository (mostly) doesn't care what kind of files you put into it, it just keeps track of changes. With the catch that diffs between binary file formats tend to be not very useful, so avoid things like Word or PDF documents, better use text based formats like Markdown, LaTeX, or HTML. There's nothing special about a resume in that regard.

If you mean to set up a website for your resume, look at Pages: https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages

**Comment by viksof** [comment]
Hi,

Thank you for your email.

Thanks for the information.

Pls understand that I am on the lookout for a job and I am finding it
difficult to find information about Github on the website of Github (i.e
https://github.com) as i believe that the website of Github is extremely
'unorganized' and I am wasting a lot of time in order to understand how to
upload the code that I have developed into my public repository on Github.

I request the concerned support staff on Github to either make the website
of Github more organized so that the information that is required by the
users of Github can be found more easily and effortlessly (in terms of
uploading project files on Github repository) or make available more
support options on the Github platform so that the user's queries can be
resolved more easily through either the Github forums or any other
discussion groups.

I cannot think about wasting my time just navigating through the
unnecessary project settings interfaces on Github when I have to think a
lot about developing, testing and deploying the code pertaining to the
project that I will be working on, rather than simply spend time trying to
understand how the configuration of project works on the Github platform.

Although, I do not intend to set up a website, I will have to upload the
'code' that I have developed in my project to the Github repository so that
I can put the corresponding website link on my resume while applying to
jobs in companies.

Can you pls let me know straight forwardly how to upload the code that I
have developed on Github public repository instead of going round about and
confusing me with unnecessary information that is not much relevant to me.

Suppose, I have to upload the code that I have developed or will be
developing in my public repository on Github and put the corresponding
website link on my resume when applying to the job Risk-Bangalore-Associate
Software Engineering as mentioned in the website link
https://higher.gs.com/roles?id=111202&title=Risk-BENGALURU-Associate-Software%20Engineering,
may I please know how can I actually upload the code that I have developed
corresponding to this job opening on my Github repository when applying to
this job?

Can you pls let me know about this clearly?

Thanks and best regards,

Vikas
